topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/tools.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
probably need to add the cummerbund tool here?

Reply:
right! Thanks @shiltemann